static/js/play.js

```diff
@@ -140,21 +139,9 @@ let app = new Vue({
 
             this.startTime += loadTime;
 
-            setMargin();
-
             //if the page's title matches that of the end article, finish the game, and submit the run
             if (page.replace("_", " ").toLowerCase() === this.endArticle.replace("_", " ").toLowerCase()) {
```

Comment:
Do we still need this? Or can we just match directly?

Reply:
Yep I think with the new API we can just match directly, will change